File: 250720/A 부터 B 까지/a-to-b.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intA = sc.nextInt();
        int intB = sc.nextInt();

        // 조건 및 반복문
        while (intA <= intB) {
            System.out.print(intA + " ");
            if (intA % 2 != 0) {
                intA *= 2;
            } else {
                intA += 3;
            }
        }
    }
}

File: 250720/A 부터 B 까지/a-to-b.py
# 입력
intA, intB = map(int, input().split())

# 첫번째 숫자 출력
print(intA, end = " ")

# 반복 및 조건문
while intA < intB :
    if intA % 2 != 0 :
        intA *= 2
        if intA > intB :
            break
        else :
            print(intA, end = " ")
    else :
        intA += 3
        if intA > intB :
            break
        else :
            print(intA, end = " ")
        
    
    